### DISCUSSION TITLE
Installation?

### DISCUSSION THREAD

**Discussion by patinas** [apirrone/Memento#73] [Q&A]
- Can I use it through WSL?
- Can make more detailed instructions on how to install - seem to have trouble with bashrc and executing memento-bg.

**Comment by apirrone** [comment]
Hi !
- I did not try through WSL, maybe you can try and report back ?
-  regarding the bashrc, you should only have write `export TESSDATA_PREFIX=/usr/share/tesseract-ocr/5/tessdata/
`  inside the file `~/.bashrc`, and `export OPENAI_API_KEY=<your key>` too if you want to use the llm capabilities. Then open a new terminal or `source ~/.bashrc`
- After running `pip install -e .` in the memento repository cloned on your machine, you should have the commands `memento-bg` and `memento-timeline` available, just type them in your terminal

Any other questions ?

**Comment by apirrone** [comment]
@patinas Did you succeed in installing Memento ?
